frontend/src/components/GeyserFirstContainer.tsx

```diff
@@ -30,6 +30,5 @@ export const GeyserFirstContainer: React.FC<Props> = () => {
 }
 
 const Container = styled.div`
-  width: 648px;
-  ${tw`text-center m-auto my-4 flex flex-wrap`}
+  ${tw`text-center m-auto my-4 flex flex-wrap w-full sm:w-648px`}
```

Comment:
maybe instead of calling it `w-648px` we could instead rename these breakpoint related sizings to `w-sm`, `w-md`, etc..?